package.json

```diff
@@ -61,7 +62,8 @@
     "build:docs": "cd docs && sphinx-versioning -g ../ -l ./conf.py build docs ../docs/_build/html",
     "deploy:docs": "cd docs && sphinx-versioning -g ../ -l ./conf.py push docs gh-pages .",
     "rebuild": "rm -rf ./node_modules && npm install && ./node_modules/.bin/webpack",
-    "prepublishOnly": "npm run lint && npm run build:prod"
+    "prepublishOnly": "npm run lint && npm run build:prod",
+    "compile-typings": "tsc"
```

Comment:
I think a more appropriate name here would `test:typings` as the use case is probably largely during dev process. It doesn't actually compile and produce anything. We can add this to the `prepublishOnly` script.